core/src/avm1/value.rs

```diff
@@ -83,6 +83,12 @@ impl<'gc> From<i32> for Value<'gc> {
     }
 }
 
+impl<'gc> From<i64> for Value<'gc> {
+    fn from(value: i64) -> Self {
+        Value::Number(value as f64)
+    }
+}
+
```

Comment:
This is a scary because i64->f64 is a potentially lossy cast. We already let `From<usize>` slip through which is also problematic, but this is especially scary if the Date behavior depends on this -- who is using this and what happens if the value is, say 2^31?

Reply:
Removed this!